src/workflow/ArcanistLandWorkflow.php

```diff
@@ -1234,6 +1256,44 @@ private function merge() {
     }
   }
 
+  private function handleDeployWinwdows() {
+    // 1. Is the current time in a blacklisted window?
+    $isInFreezeWindow = $this->isInDeployWindows($this->deployWindowsBlacklist, time());
+
+    /**
+     * If in window, depending on the strategy:
+     *  strategy=warn: ask user to confirm or exit
+     *  strategy=reject: simply reject
```

Comment:
I'd like to push back on even allowing `strategy=reject` as an option. Seems rather draconian. For our purposes, warn seems sufficient.

Reply:
That's fair. I removed it for now